<!DOCTYPE html>
<html lang="en">

<head>

<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />

<meta name="description" content="Name of your web site">
<meta name="author" content="Marketify">

<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

<title>Colby T. Ford, Ph.D.</title>

<!-- STYLES -->
<link href="https://fonts.googleapis.com/css?family=Montserrat:400,400i,500,500i,600,600i,700,700i,800,800i,900,900i" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,400i,600,600i,700,700i,800,800i" rel="stylesheet">
<link rel="stylesheet" type="text/css" href="css/plugins.css" />
<link rel="stylesheet" type="text/css" href="css/style.css" />
<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
<link rel="manifest" href="/site.webmanifest">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
<!--[if lt IE 9]> <script type="text/javascript" src="js/modernizr.custom.js"></script> <![endif]-->
<!-- /STYLES -->

</head>

<body>

<!-- WRAPPER ALL -->
<div class="edina_tm_wrapper_all">

	<div id="edina_tm_popup_blog">
		<div class="container">
			<div class="inner_popup scrollable"></div>
		</div>
		<span class="close"><a href="#"></a></span>
	</div>

    <!-- HEADER -->
    <header class="edina_tm_header">
		<div class="edina_tm_navigation_wrap">
			<div class="container">
				<div class="navigation_inner">
					<div class="logo">
						<!--<a class="dark_logo" href="#">Colby T. Ford, Ph.D.</a>-->
						<a class="dark_logo" href="#"><img src="img/svg/title.svg" alt="" width="200px" /></a>
					</div>
					<div class="nav_list_wrap">
						<div class="menu">
							<ul class="anchor_nav">
								<li><a href="#home">Home</a></li>
								<li><a href="#about">About</a></li>
								<li><a href="#skills">Skills</a></li>
								<!--<li><a href="#portfolio">Portfolio</a></li>-->
								<!--<li><a href="#testimonials">Testimonials</a></li>-->
								<li><a href="#research">Research</a></li>
								<li><a href="https://colbyford.medium.com/" target="_blank">Blog</a></li>
								<li><a href="#contact">Contact</a></li>
							</ul>
						</div>
						<div class="social_icons_wrap">
								<ul>
										<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.2s"><a href="https://www.github.com/colbyford" title="GitHub" target="_blank"><i class="fab fa-github"></i></a></li>
										<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.4s"><a href="https://www.linkedin.com/in/colbyford" title="LinkedIn" target="_blank"><i class="fab fa-linkedin"></i></a></li>
										<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.6s"><a href="https://scholar.google.com/citations?user=zetDB4EAAAAJ&hl=en&authuser=1" title="Google Scholar" target="_blank"><i class="fas fa-graduation-cap"></i></a></li>
										<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.8s"><a href="https://www.researchgate.net/profile/Colby_Ford" title="ResearchGate" target="_blank"><i class="fab fa-researchgate"></i></a></li>
										<!-- <li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="1.0s"><a href="https://www.instagram.com/c01.by/" title="Instagram" target="_blank"><i class="fab fa-instagram"></i></a></li> -->
										<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="1.2s"><a href="mailto:colby.ford@uncc.edu" title="Email" target="_blank"><i class="fas fa-envelope"></i></a></li>
										<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="1.4s"><a href="tel:18287738009" title="Phone Number" target="_blank"><i class="fas fa-phone"></i></a></li>
									</ul>
						</div>
					</div>
					<div class="edina_tm_trigger">
						<div class="hamburger hamburger--collapse-r">
							<div class="hamburger-box">
								<div class="hamburger-inner"></div>
							</div>
						</div>
					</div>
				</div>
			</div>
		</div>
   		<div class="edina_tm_mobile_menu_wrap">
   			<div class="mob_menu">
				<ul class="anchor_nav">
					<li><a href="#home">Home</a></li>
					<li><a href="#about">About</a></li>
					<li><a href="#skills">Skills</a></li>
					<!--<li><a href="#portfolio">Portfolio</a></li>-->
					<!--<li><a href="#testimonials">Testimonials</a></li>-->
					<li><a href="#research">Research</a></li>
					<li><a href="#contact">Contact</a></li>
				</ul>
			</div>
		</div>
    </header>
    <!-- /HEADER -->

    <!-- CONTENT -->
	<div class="edina_tm_content">

		<!-- HERO -->
		<div class="edina_tm_section" id="home">
			<div class="edina_tm_hero_header">
				<div class="edina_tm_universal_box_wrap">
					<div class="bg_wrap">
						<div class="overlay_image hero jarallax" data-speed="0"></div>
						<div class="overlay_video"></div>
						<div class="overlay_color hero"></div>
					</div>
					<div class="content hero">
						<div class="container hero">
							<div class="hero_title">
								<p class="first">Hi! I'm Colby...<br/><span class="edina_tm_animation_text_word"></span></p>
							</div>
						</div>
						<div class="edina_tm_discover_wrap anchor">
							<span>
								<a href="#about">Discover</a>
							</span>
						</div>
					</div>
				</div>
			</div>
		</div>
		<!-- /HERO -->

		<!-- ABOUT -->
		<div class="edina_tm_section" id="about">
			<div class="container">
				<div class="edina_tm_main_title_holder_wrap about">
					<div class="number_wrap">
						<span>01</span>
					</div>
					<div class="title_wrap">
						<span>About Me</span>
					</div>
				</div>
				<div class="edina_tm_about_wrap homepage_second">
					<div class="author_wrap">
						<div class="leftbox"></div>
						<div class="rightbox">
							<div class="name_holder">
								<h3>I'm Colby T. Ford, Ph.D.</h3>
							</div>
							<div class="definition">
								<p>I'm a Cloud Genomics and AI Architect. I have a passion for Mathematics and Bioinformatics research. So, I use Machine Learning and Cloud Computing to help along the way.</p>
								<p>I am the founder of <a href="https://www.tuple.xyz" target="_blank" style="color: #e95523">&lt;Tuple&gt;</a>, a genomics-focused Microsoft and Databricks partner consulting firm. Also, I'm currently a visiting scholar in the University of North Carolina at Charlotte's <a href="https://cipher.charlotte.edu" target="_blank" style="color: #e95523">CIPHER</a>a research center where I do a lot of protein modeling work in infectious diseases (and I teach a graduate course in Cloud Computing).</p>
								<p>Originally from Lenoir, North Carolina, I studied Applied Mathematics, Data Science, and Computational Biology at the University of North Carolina at Charlotte. I am currently based in Charlotte, North Carolina.</p>
							</div>
							<div class="button" style="text-align:center">
									<a href="ColbyFord_CV.pdf" target="_blank"><span>Download CV</span></a>
							</div>
							<br>
							<div class="sharebox" style="text-align:center">
								<ul>
									<li class="wow fadeIn" data-wow-duration="1.2s"><span>Connect:</span></li>
									<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.2s"><a href="https://www.github.com/colbyford" title="GitHub" target="_blank"><i class="fab fa-github"></i></a></li>
									<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.4s"><a href="https://www.linkedin.com/in/colbyford" title="LinkedIn" target="_blank"><i class="fab fa-linkedin"></i></a></li>
									<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.6s"><a href="https://scholar.google.com/citations?user=zetDB4EAAAAJ&hl=en&authuser=1" title="Google Scholar" target="_blank"><i class="fas fa-graduation-cap"></i></a></li>
									<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.8s"><a href="https://www.researchgate.net/profile/Colby_Ford" title="ResearchGate" target="_blank"><i class="fab fa-researchgate"></i></a></li>
									<!-- <li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="1.0s"><a href="https://www.instagram.com/c01.by/" title="Instagram" target="_blank"><i class="fab fa-instagram"></i></a></li> -->
									<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="1.2s"><a href="mailto:colby.ford@uncc.edu" title="Email" target="_blank"><i class="fas fa-envelope"></i></a></li>
									<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="1.4s"><a href="tel:18287738009" title="Phone Number" target="_blank"><i class="fas fa-phone"></i></a></li>
								</ul>
							</div>
						</div>
					</div>
				</div>
			</div>
		</div>
		<!-- /ABOUT -->

		<!-- SERVICES -->
		<div class="edina_tm_section" id="skills">
			<div class="edina_tm_services_total_wrap">
				<div class="container">
					<div class="edina_tm_main_title_holder_wrap">
						<div class="number_wrap">
							<span>02</span>
						</div>
						<div class="title_wrap">
							<span>Skills</span>
						</div>
					</div>
					<div class="edina_tm_services_wrap">
						<div class="edina_tm_list_wrap" data-column="3" data-space="70">
							<ul class="total">
								<li class="wow fadeIn" data-wow-duration="1.2s">
									<div class="inner_list">
										<div class="service_icon">
											<img class="svg" src="img/svg/abacus.svg" alt="" />
										</div>
										<div class="service_title">
											<h3>Mathematics</h3>
										</div>
										<div class="service_definition">
											<p>Statistical Modeling<br>Bayesian Nets<br>Operations Research<br>Numerical Logic</p>
										</div>
										<span class="first"></span>
										<span class="second"></span>
									</div>
								</li>
								<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.2s">
									<div class="inner_list">
										<div class="service_icon">
											<img class="svg" src="img/svg/artificial-intelligence.svg" alt="" />
										</div>
										<div class="service_title">
											<h3>Machine Learning</h3>
										</div>
										<div class="service_definition">
											<p>Algorithm Design<br>Natural Language Processing<br>Deep Learning<br>Artificial Intelligence</p>
										</div>
										<span class="first"></span>
										<span class="second"></span>
									</div>
								</li>
								<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.4s">
									<div class="inner_list">
										<div class="service_icon">
											<img class="svg" src="img/svg/dna.svg" alt="" />
										</div>
										<div class="service_title">
											<h3>Computational Biology</h3>
										</div>
										<div class="service_definition">
											<p>Human Genomics<br>Phylogenetics<br>Molecular Sequence Analysis<br>Infectious Diseases</p>
										</div>
										<span class="first"></span>
										<span class="second"></span>
									</div>
								</li>
								<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.6s">
									<div class="inner_list">
										<div class="service_icon">
											<img class="svg" src="img/svg/hardware.svg" alt="" />
										</div>
										<div class="service_title">
											<h3>Distributed Computing</h3>
										</div>
										<div class="service_definition">
											<p>Apache Spark (Databricks)<br>Cluster Computing (MPI/SNOW)<br>GPU-Based Processing (CUDA)<br>High Performance Computing</p>
										</div>
										<span class="first"></span>
										<span class="second"></span>
									</div>
								</li>
								<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.8s">
									<div class="inner_list">
										<div class="service_icon">
											<img class="svg" src="img/svg/code.svg" alt="" />
										</div>
										<div class="service_title">
											<h3>Programming</h3>
										</div>
										<div class="service_definition">
											<p>R/SparkR<br>Python/PySpark<br>SQL<br>Visual Basic</p>
										</div>
										<span class="first"></span>
										<span class="second"></span>
									</div>
								</li>
								<li class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="1s">
									<div class="inner_list">
										<div class="service_icon">
											<img class="svg" src="img/svg/computer-graphic.svg" alt="" />
										</div>
										<div class="service_title">
											<h3>Web and Design</h3>
										</div>
										<div class="service_definition">
											<p>HTML5+CSS<br>LaTeX<br>API Development<br>Visualization</p>
										</div>
										<span class="first"></span>
										<span class="second"></span>
									</div>
								</li>
							</ul>
						</div>
					</div>
				</div>
			</div>
		</div>
		<!-- /SERVICES -->

		<!-- PORTFOLIO -->
		<!--<div class="edina_tm_section" id="portfolio">
			<div class="container">
				<div class="edina_tm_portfolio_wrap">
					<div class="edina_tm_main_title_holder_wrap portfolio">
						<div class="number_wrap">
							<span>03</span>
						</div>
						<div class="title_wrap">
							<span>My Research</span>
						</div>
					</div>
					<ul class="edina_tm_portfolio_filter">
						<li><a href="#" class="current" data-filter="*">All</a></li>
						<li><a href="#" data-filter=".design">Design</a></li>
						<li><a href="#" data-filter=".photography">Photography</a></li>
						<li><a href="#" data-filter=".development">Development</a></li>
					</ul>
					<ul class="edina_tm_portfolio_list gallery_zoom">
						<li class="design">
							<div class="list_inner">
								<div class="image_wrap">
									<img src="img/portfolio/600x600.jpg" alt="" />
									<div class="main_image"></div>
								</div>
								<div class="definition_portfolio">
									<span class="first"><a class="zoom" href="img/portfolio/5.jpg">Title of project</a></span>
									<span class="second">Category</span>
								</div>
							</div>
						</li>
						<li class="photography">
							<div class="list_inner">
								<div class="image_wrap">
									<img src="img/portfolio/600x600.jpg" alt="" />
									<div class="main_image"></div>
								</div>
								<div class="definition_portfolio">
									<span class="first"><a class="zoom" href="img/portfolio/2.jpg">Title of project</a></span>
									<span class="second">Category</span>
								</div>
							</div>
						</li>
						<li class="development">
							<div class="list_inner">
								<div class="image_wrap">
									<img src="img/portfolio/600x600.jpg" alt="" />
									<div class="main_image"></div>
								</div>
								<div class="definition_portfolio">
									<span class="first"><a class="zoom" href="img/portfolio/3.jpg">Title of project</a></span>
									<span class="second">Category</span>
								</div>
							</div>
						</li>
						<li class="photography">
							<div class="list_inner">
								<div class="image_wrap">
									<img src="img/portfolio/600x600.jpg" alt="" />
									<div class="main_image"></div>
								</div>
								<div class="definition_portfolio">
									<span class="first"><a class="zoom" href="img/portfolio/4.jpg">Title of project</a></span>
									<span class="second">Category</span>
								</div>
							</div>
						</li>
						<li class="design">
							<div class="list_inner">
								<div class="image_wrap">
									<img src="img/portfolio/600x600.jpg" alt="" />
									<div class="main_image"></div>
								</div>
								<div class="definition_portfolio">
									<span class="first"><a class="zoom" href="img/portfolio/6.jpg">Title of project</a></span>
									<span class="second">Category</span>
								</div>
							</div>
						</li>
						<li class="photography">
							<div class="list_inner">
								<div class="image_wrap">
									<img src="img/portfolio/600x600.jpg" alt="" />
									<div class="main_image"></div>
								</div>
								<div class="definition_portfolio">
									<span class="first"><a class="zoom" href="img/portfolio/7.jpg">Title of project</a></span>
									<span class="second">Category</span>
								</div>
							</div>
						</li>
					</ul>
				</div>
			</div>
		</div>-->
		<!-- /PORTFOLIO -->

		<!-- TESTIMONIALS -->
		<!--<div class="edina_tm_section" id="testimonials">
			<div class="edina_tm_testimonial_wrap">
				<div class="edina_tm_universal_box_wrap">
					<div class="bg_wrap hero">
						<div class="overlay_image testimonial jarallax" data-speed="0"></div>
						<div class="overlay_color testimonial"></div>
					</div>
					<div class="content testimonial">
						<div class="edina_tm_main_title_holder_wrap testimonial">
							<div class="number_wrap">
								<span>04</span>
							</div>
							<div class="title_wrap">
								<span>Testimonials</span>
							</div>
						</div>
						<div class="container">
							<div class="carousel_wrap">
								<ul class="owl-carousel">
									<li class="item">
										<div class="inner">
											<div class="image_holder">
												<img src="img/clients/1.jpg" alt="" />
											</div>
											<div class="definition">
												<p>Edina team are easy to work with and helped me make amazing websites in a short amount of time. Thanks guys for all your hard work.</p>
											</div>
											<div class="svg_wrap">
												<i class="xcon-quote-left"></i>
											</div>
											<div class="name_holder_wrap">
												<span class="name">Antonio Baraley</span>
												<span class="job">CEO Founder</span>
											</div>
										</div>
									</li>
									<li class="item">
										<div class="inner">
											<div class="image_holder">
												<img src="img/clients/2.jpg" alt="" />
											</div>
											<div class="definition">
												<p>We were looking for a logo with a touch of modernism. Edina grasped our needs and produced a stunning design.</p>
											</div>
											<div class="svg_wrap">
												<i class="xcon-quote-left"></i>
											</div>
											<div class="name_holder_wrap">
												<span class="name">Calena Gomez</span>
												<span class="job">AOC Designer</span>
											</div>
										</div>
									</li>
									<li class="item">
										<div class="inner">
											<div class="image_holder">
												<img src="img/clients/3.jpg" alt="" />
											</div>
											<div class="definition">
												<p>Awesome to work with Edina. Incredibly organized, easy to communicate with, responsive with next iterations.</p>
											</div>
											<div class="svg_wrap">
												<i class="xcon-quote-left"></i>
											</div>
											<div class="name_holder_wrap">
												<span class="name">Torren Winston</span>
												<span class="job">Photographer</span>
											</div>
										</div>
									</li>
								</ul>
							</div>
						</div>
					</div>
				</div>
			</div>
		</div>-->
		<!-- /TESTIMONIALS -->

		<!-- NEWS -->
		<div class="edina_tm_section" id="research">
			<div class="edina_tm_news_wrap">
				<div class="container">
					<div class="edina_tm_main_title_holder_wrap news">
						<div class="number_wrap">
							<span>03</span>
						</div>
						<div class="title_wrap">
							<span>Research</span>
						</div>
					</div>

					<div class="edina_tm_list_wrap blog_list" data-column="3" data-space="30">
						<ul class="total">
							<li class="wow fadeInUp" data-wow-duration="1.2s">
								<div class="inner_list">
									<div class="image_wrap">
										<img class="small" src="research/img/zebrafish_mecp2.PNG" alt="" />
										<img class="big" src="research/img/zebrafish_mecp2.PNG" alt="" width="50%"/>
									</div>
									<div class="definitions_wrap">
										<div class="date_wrap">
											<p>Genomics, RNA-seq, Zebrafish</p>
										</div>
										<div class="title_holder">
											<h3><i>mecp2</i> function in zebrafish</i></h3>
										</div>
										<div class="definition">
											<p>MeCP2's function as a transcriptional regulator is conserved across vertebrates and supports using zebrafish to complement mouse modeling in elucidating these conserved mechanisms.</p>
										</div>
										<div class="full_def">
											<p>Rett syndrome (RTT), a human neurodevelopmental disorder characterized by severe cognitive and motor impairments, is caused by dysfunction of the conserved transcriptional regulator Methyl-CpG-binding protein 2 (MECP2). Genetic analyses in mouse Mecp2 mutants, which exhibit key features of human RTT, have been essential for deciphering the mechanisms of MeCP2 function; nonetheless, our understanding of these complex mechanisms is incomplete. Zebrafish mecp2 mutants exhibit mild behavioral deficits but have not been analyzed in depth. Here, we combine transcriptomic and behavioral assays to assess baseline and stimulus-evoked motor responses and sensory filtering in zebrafish mecp2 mutants from 5 to 7 days post-fertilization (dpf). We show that zebrafish mecp2 function is required for normal thigmotaxis but is dispensable for gross movement, acoustic startle response, and sensory filtering (habituation and sensorimotor gating), and reveal a previously unknown role for mecp2 in behavioral responses to visual stimuli. RNA-seq analysis identified a large gene set that requires mecp2 function for correct transcription at 4 dpf, and pathway analysis revealed several pathways that require MeCP2 function in both zebrafish and mammals. These findings show that MeCP2's function as a transcriptional regulator is conserved across vertebrates and supports using zebrafish to complement mouse modeling in elucidating these conserved mechanisms.</p>
										</div>
										<div class="edina_tm_popup_share_wrap">
												<ul>
													<li><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ajmg.b.32981" target="_blank">Paper</a></li>
													<li><a href="https://github.com/colbyford/zebrafish_mecp2_expression" target="_blank">GitHub</a></li>
												</ul>
											</div>
										<div class="read_more">
											<a href="#"><span>Read More</span></a>
										</div>
									</div>
								</div>
							</li>
							<li class="wow fadeInUp" data-wow-duration="1.2s">
								<div class="inner_list">
									<div class="image_wrap">
										<img class="small" src="research/img/brocho_organoids.PNG" alt="" />
										<img class="big" src="research/img/brocho_organoids.PNG" alt="" width="50%"/>
									</div>
									<div class="definitions_wrap">
										<div class="date_wrap">
											<p>Genomics, Oncology, Antibodies</p>
										</div>
										<div class="title_holder">
											<h3>Bronchioalveolar organoids in ADC toxicity</h3>
										</div>
										<div class="definition">
											<p>Bronchioalveolar organoids: A preclinical tool to screen toxicity associated with antibody-drug conjugates</p>
										</div>
										<div class="full_def">
											<p>Despite extensive preclinical testing, cancer therapeutics can result in unanticipated toxicity to non-tumor tissue in patients. These toxicities may pass undetected in preclinical experiments due to modeling limitations involving poor biomimicry of 2-dimensional in vitro cell cultures and due to lack of interspecies translatability in in vivo studies. Instead, primary cells can be grown into miniature 3-dimensional structures that recapitulate morphological and functional aspects of native tissue, termed “organoids.” Here, human bronchioalveolar organoids grown from primary alveolar epithelial cells were employed to model lung epithelium and investigate off-target toxicities associated with antibody-drug conjugates (ADCs). ADCs with three different linker-payload combinations (mafodotin, vedotin, and deruxtecan) were tested in bronchioalveolar organoids generated from human, rat, and nonhuman primate lung cells. Organoids demonstrated antibody uptake and changes in viability in response to ADC exposure that model in vivo drug sensitivity. RNA sequencing identified inflammatory activation in bronchioalveolar cells in response to deruxtecan. Future studies will explore specific cell populations involved in interstitial lung disease and incorporate immune cells to the culture.</p>
										</div>
										<div class="edina_tm_popup_share_wrap">
											<ul>
												<li><a href="https://www.sciencedirect.com/science/article/pii/S0041008X2400084X" target="_blank">Paper</a></li>
											</ul>
											</div>
										<div class="read_more">
											<a href="#"><span>Read More</span></a>
										</div>
									</div>
								</div>
							</li>
							<li class="wow fadeInUp" data-wow-duration="1.2s">
								<div class="inner_list">
									<div class="image_wrap">
										<img class="small" src="research/img/n_cytokine.PNG" alt="" />
										<img class="big" src="research/img/n_cytokine.PNG" alt="" width="50%"/>
									</div>
									<div class="definitions_wrap">
										<div class="date_wrap">
											<p>Genomics, Machine Learning, SARS-CoV-2</p>
										</div>
										<div class="title_holder">
											<h3>Human Cytokine and SARS-CoV-2 N interactivity</h3>
										</div>
										<div class="definition">
											<p>Human Cytokine and Coronavirus Nucleocapsid Protein Interactivity Using Large-Scale Virtual Screens</p>
										</div>
										<div class="full_def">
											<p>Understanding the interactions between SARS-CoV-2 and the human immune system is paramount to the characterization of novel variants as the virus co-evolves with the human host. In this study, we employed state-of-the-art molecular docking tools to conduct large-scale virtual screens, predicting the binding affinities between 64 human cytokines against 17 nucleocapsid proteins from six betacoronaviruses. Our comprehensive in silico analyses reveal specific changes in cytokine-nucleocapsid protein interactions, shedding light on potential modulators of the host immune response during infection. These findings offer valuable insights into the molecular mechanisms underlying viral pathogenesis and may guide the future development of targeted interventions. This manuscript serves as insight into the comparison of deep learning based AlphaFold2-Multimer and the semi-physicochemical based HADDOCK for protein-protein docking. We show the two methods are complementary in their predictive capabilities. We also introduce a novel algorithm for rapidly assessing the binding interface of protein-protein docks using graph edit distance: graph-based interface residue assessment function (GIRAF). The high-performance computational framework presented here will not only aid in accelerating the discovery of effective interventions against emerging viral threats, but extend to other applications of high throughput protein-protein screens.</p>
										</div>
										<div class="edina_tm_popup_share_wrap">
											<ul>
												<li><a href="https://www.biorxiv.org/content/10.1101/2023.11.28.569056" target="_blank">Paper</a></li>
												<li><a href="https://github.com/tuplexyz/SARS-CoV-2_N-Cytokine_Docking" target="_blank">GitHub Link</a></li>
											</ul>
											</div>
										<div class="read_more">
											<a href="#"><span>Read More</span></a>
										</div>
									</div>
								</div>
							</li>
						</ul>
					</div>

					<div class="edina_tm_list_wrap blog_list" data-column="3" data-space="30">
						<ul class="total">
							<li class="wow fadeInUp" data-wow-duration="1.2s">
								<div class="inner_list">
									<div class="image_wrap">
										<img class="small" src="research/img/plasmodium_comparisons.PNG" alt="" />
										<img class="big" src="research/img/plasmodium_comparisons.PNG" alt="" width="50%"/>
									</div>
									<div class="definitions_wrap">
										<div class="date_wrap">
											<p>Genomics, scRNA-seq, Plasmodium</p>
										</div>
										<div class="title_holder">
											<h3>Comparative transcriptomics of <i>Plasmodium vivax</i></h3>
										</div>
										<div class="definition">
											<p>Comparative transcriptomics reveal differential gene expression among <i>Plasmodium vivax</i> geographical isolates and implications on erythrocyte invasion mechanisms</p>
										</div>
										<div class="full_def">
											<p>The documentation of <i>Plasmodium vivax</i> malaria across Africa, especially in regions where Duffy negatives are dominant, suggests possibly alternative erythrocyte invasion mechanisms. While the transcriptomes of the Southeast Asian and South American P. vivax are well documented, the gene expression profile of P. vivax in Africa is unclear. In this study, we examined the expression of 4,404 gene transcripts belong to 12 functional groups and 43 erythrocyte binding gene candidates in Ethiopian isolates and compared them with the Cambodian and Brazilian P. vivax transcriptomes. Overall, there were 10–26% differences in the gene expression profile amongst geographical isolates, with the Ethiopian and Cambodian P. vivax being most similar. Majority of the gene transcripts involved in protein transportation, housekeeping, and host interaction were highly transcribed in the Ethiopian isolates. Members of the reticulocyte binding protein PvRBP2a and PvRBP3 expressed six-fold higher than Duffy binding protein PvDBP1 and 60-fold higher than PvEBP/DBP2 in the Ethiopian isolates. Other genes including PvMSP3.8, PvMSP3.9, PvTRAG2, PvTRAG14, and PvTRAG22 also showed relatively high expression. Differential expression patterns were observed among geographical isolates, e.g., PvDBP1 and PvEBP/DBP2 were highly expressed in the Cambodian but not the Brazilian and Ethiopian isolates, whereas PvRBP2a and PvRBP2b showed higher expression in the Ethiopian and Cambodian than the Brazilian isolates. Compared to Pvs25, gametocyte genes including PvAP2-G, PvGAP (female gametocytes), and Pvs47 (male gametocytes) were highly expressed across geographical samples.</p>
										</div>
										<div class="edina_tm_popup_share_wrap">
											<ul>
												<li><a href="https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0011926" target="_blank">Paper</a></li>
												<li><a href="https://github.com/colbyford/vivax_transcriptome_comparisons" target="_blank">GitHub</a></li>
											</ul>
											</div>
										<div class="read_more">
											<a href="#"><span>Read More</span></a>
										</div>
									</div>
								</div>
							</li>
							<li class="wow fadeInUp" data-wow-duration="1.2s">
								<div class="inner_list">
									<div class="image_wrap">
										<img class="small" src="research/img/luidia_sh_network.PNG" alt="" />
										<img class="big" src="research/img/luidia_sh_network.PNG" alt="" width="50%"/>
										<!--<div class="news_image" data-url="research/img/strainhub.png"></div>-->
										<!--<a class="link_news" href="index.html"></a>-->
									</div>
									<div class="definitions_wrap">
										<div class="date_wrap">
											<p>Genomics, Phlogenetics, Echinodermata</p>
										</div>
										<div class="title_holder">
											<h3>Occurrence of the starfish <i>Luidia magnifica</i> in the Mexican Pacific</h3>
										</div>
										<div class="definition">
											<p>Occurrence of the Indo-West Pacific starfish <i>Luidia magnifica</i> (Echinodermata: Asteroidea) in the Mexican Pacific and a possible introduction to the Caribbean region</p>
										</div>
										<div class="full_def">
											<p>The starfish Luidia magnifica (Asteroidea: Paxillosida: Luidiidae) is a multiradiate starfish that occurs throughout the Indo-Pacific. Since the description of L. magnifica in 1906 by Fisher, very few observations of this species have been documented. We present new shallow-water records for L. magnifica in the Tropical Eastern Pacific obtained between 2017 and 2021. We collected a total of 15 specimens of L. magnifica in the bay of Manzanillo, Colima, in the central nearshore Mexican Pacific. We provide morphological description of specimens collected and additional ecological data. We used molecular techniques to verify species identification. This record represents a considerable expansion of the known distribution range of L. magnifica and provides the first evidence of the presence of this species in the Tropical Eastern Pacific. The presence of L. magnifica larvae from the Caribbean coast of Panama was also identified from DNA sequencing of larvae left unindentified in previous reports. This result indicates that it is possible that the L. magnifica also occurs in the Carribbean. Because observations of the starfish in the Mexican Pacific have been recent, the presence of larvae in the Caribbean could be a recent introduction through the Panama Canal from the Eastern Pacific to the Western Atlantic.</p>
										</div>
										<div class="edina_tm_popup_share_wrap">
												<ul>
													<li><a href="https://link.springer.com/article/10.1007/s12526-023-01397-2" target="_blank">Paper</a></li>
													<li><a href="https://github.com/colbyford/luidia_magnifica_in_mexican_pacific" target="_blank">GitHub</a></li>
												</ul>
											</div>
										<div class="read_more">
											<a href="#"><span>Read More</span></a>
										</div>
									</div>
								</div>
							</li>
							<li class="wow fadeInUp" data-wow-duration="1.2s">
								<div class="inner_list">
									<div class="image_wrap">
										<img class="small" src="research/img/sand_dollar_tree.PNG" alt="" />
										<img class="big" src="research/img/sand_dollar_tree.PNG" alt="" width="100%"/>
									</div>
									<div class="definitions_wrap">
										<div class="date_wrap">
											<p>Genomics, Phylogenetics, Echinodermata</p>
										</div>
										<div class="title_holder">
											<h3>Phylogeny, ancestral ranges and reclassification of sand dollars</h3>
										</div>
										<div class="definition">
											<p>Phylogeny, ancestral ranges and reclassification of sand dollars</p>
										</div>
										<div class="full_def">
											<p>Classification of the Class Echinoidea is under significant revision in light of emerging molecular phylogenetic evidence. In particular, the sister-group relationships within the superorder Luminacea (Echinoidea: Irregularia) have been considerably updated. However, the placement of many families remains largely unresolved due to a series of incongruent evidence obtained from morphological, paleontological, and genetic data for the majority of extant representatives. In this study, we investigated the phylogenetic relationships of 25 taxa, belonging to eleven luminacean families. We proposed three new superfamilies: Astriclypeoidea, Mellitoidea, and Taiwanasteroidea (including Dendrasteridae, Taiwanasteridae, Scutellidae, and Echinarachniidae), instead of the currently recognized superfamily Scutelloidea Gray, 1825. In light of the new data obtained from ten additional species, the historical biogeography reconstructed shows that the tropical western Pacific and eastern Indian Oceans are the cradle for early sand dollar diversification. Hothouse conditions during the late Cretaceous and early Paleogene were coupled with diversification events of major clades of sand dollars. We also demonstrate that Taiwan fauna can play a key role in terms of understanding the major Cenozoic migration and dispersal events in the evolutionary history of Luminacea.</p>
										</div>
										<div class="edina_tm_popup_share_wrap">
												<ul>
													<li><a href="https://www.nature.com/articles/s41598-023-36848-0" target="_blank">Paper</a></li>
													<li><a href="https://github.com/colbyford/reclassification-of-sand-dollars" target="_blank">GitHub Link</a></li>
												</ul>
											</div>
										<div class="read_more">
											<a href="#"><span>Read More</span></a>
										</div>
									</div>
								</div>
							</li>
						</ul>
					</div>

					<div class="edina_tm_list_wrap blog_list" data-column="3" data-space="30">
						<ul class="total">
							<li class="wow fadeInUp" data-wow-duration="1.2s">
								<div class="inner_list">
									<div class="image_wrap">
										<img class="small" src="research/img/xbb15_rbd.png" alt="" />
										<img class="big" src="research/img/xbb15_rbd.png" alt="" width="50%"/>
										<!--<div class="news_image" data-url="research/img/strainhub.png"></div>-->
										<!--<a class="link_news" href="index.html"></a>-->
									</div>
									<div class="definitions_wrap">
										<div class="date_wrap">
											<p>Genomics, SARS-CoV-2, Machine Learning</p>
										</div>
										<div class="title_holder">
											<h3>SARS-CoV-2 XBB.1.5 RBD-Antibody Predictions</h3>
										</div>
										<div class="definition">
											<p>Predicting changes in neutralizing antibody activity for SARS-CoV-2 XBB.1.5 using <i>in silico</i> protein modeling..</p>
										</div>
										<div class="full_def">
												<p>The SARS-CoV-2 variant XBB.1.5 is of concern as it has high transmissibility. XBB.1.5 currently accounts for upwards of 30% of new infections in the United States. One year after our group published the predicted structure of the Omicron (B.1.1.529) variant's receptor binding domain and antibody binding affinity, we return to investigate the new mutations seen in XBB.1.5 which is a descendant of Omicron. Using <i>in silico</i> modeling approaches against newer neutralizing antibodies that are shown effective against B.1.1.529, we posit the immune consequences of XBB.1.5's mutations and show that there is no statistically significant difference in overall antibody evasion when comparing to the B.1.1.529 and other related variants (e.g. BJ.1 and BM.1.1.1). However, noticeable changes in neutralizing activity were seen due to specific amino acid changes of interest in the newer variants.</p>
										</div>
										<div class="edina_tm_popup_share_wrap">
												<ul>
													<li><a href="https://www.frontiersin.org/articles/10.3389/fviro.2023.1172027/full" target="_blank">Paper</a></li>
 													<!-- <li><a href="https://www.biorxiv.org/content/10.1101/2023.02.10.528025" target="_blank">Preprint</a></li> -->
													<li><a href="https://github.com/colbyford/SARS-CoV-2_XBB.1.5_Spike-RBD_Predictions" target="_blank">GitHub Link</a></li>
												</ul>
											</div>
										<div class="read_more">
											<a href="#"><span>Read More</span></a>
										</div>
									</div>
								</div>
							</li>
							<li class="wow fadeInUp" data-wow-duration="1.2s">
								<div class="inner_list">
									<div class="image_wrap">
										<img class="small" src="research/img/genomics_in_azure.png" alt="" />
										<img class="big" src="research/img/genomics_in_azure.png" alt="" width="50%"/>
										<!--<div class="news_image" data-url="research/img/strainhub.png"></div>-->
										<!--<a class="link_news" href="index.html"></a>-->
									</div>
									<div class="definitions_wrap">
										<div class="date_wrap">
											<p>Genomics, Cloud, Machine Learning</p>
										</div>
										<div class="title_holder">
											<h3><i>Genomics in the Azure Cloud</i> Book</h3>
										</div>
										<div class="definition">
											<p>Building a cloud genomics architecture in Azure using enterprise-grade platform services.</p>
										</div>
										<div class="full_def">
												<p>This practical guide bridges the gap between general cloud computing architecture in Microsoft Azure and scientific computing for bioinformatics and genomics. You'll get a solid understanding of the architecture patterns and services that are offered in Azure and how they might be used in your bioinformatics practice. You'll get code examples that you can reuse for your specific needs. And you'll get plenty of concrete examples to illustrate how a given service is used in a bioinformatics context.</p>
										</div>
										<div class="edina_tm_popup_share_wrap">
												<ul>
													<li><a href="https://www.oreilly.com/library/view/genomics-in-the/9781098139032/" target="_blank">Buy now on O'Reilly.com</a></li>
													<li><a href="https://www.amazon.com/Genomics-Azure-Cloud-Bioinformatics-Enterprise-Grade/dp/1098139046" target="_blank">Buy now on Amazon.com</a></li>
													<li><a href="https://www.target.com/p/genomics-in-the-azure-cloud-by-colby-ford-paperback/-/A-87567449" target="_blank">Buy now on Target.com</a></li>
												</ul>
											</div>
										<div class="read_more">
											<a href="#"><span>Read More</span></a>
										</div>
									</div>
								</div>
							</li>
							<li class="wow fadeInUp" data-wow-duration="1.2s">
								<div class="inner_list">
									<div class="image_wrap">
										<img class="small" src="research/img/b11529_rbd.png" alt="" />
										<img class="big" src="research/img/b11529_rbd.png" alt="" width="50%"/>
										<!--<div class="news_image" data-url="research/img/strainhub.png"></div>-->
										<!--<a class="link_news" href="index.html"></a>-->
									</div>
									<div class="definitions_wrap">
										<div class="date_wrap">
											<p>Genomics, SARS-CoV-2, Machine Learning</p>
										</div>
										<div class="title_holder">
											<h3>SARS-CoV-2 Omicron RBD Predictions</h3>
										</div>
										<div class="definition">
											<p>Deep learning-based predictions of the SARS-CoV-2 Omicron (B.1.1.529) variant receptor binding domain with neutralizing antibodies.</p>
										</div>
										<div class="full_def">
												<p>The genome of the SARS-CoV-2 Omicron variant (B.1.1.529) was released on November 22, 2021, which has caused a flurry of media attention due the large number of mutations it contains. These raw data have spurred questions around vaccine efficacy. Given that neither the structural information nor the experimentally-derived antibody interaction of this variant are available, we have turned to predictive computational methods to model the mutated structure of the spike protein's receptor binding domain and posit potential changes to vaccine efficacy. In this study, we predict some structural changes in the receptor-binding domain that may reduce antibody interaction without completely evading existing neutralizing antibodies (and therefore current vaccines).</p>
										</div>
										<div class="edina_tm_popup_share_wrap">
												<ul>
													<li><a href="https://www.frontiersin.org/articles/10.3389/fviro.2022.830202/full" target="_blank">Paper</a></li>
													<!-- <li><a href="https://www.biorxiv.org/content/10.1101/2021.12.03.471024" target="_blank">Preprint</a></li> -->
													<li><a href="https://github.com/colbyford/SARS-CoV-2_B.1.1.529_Spike-RBD_Predictions" target="_blank">GitHub Link</a></li>
												</ul>
											</div>
										<div class="read_more">
											<a href="#"><span>Read More</span></a>
										</div>
									</div>
								</div>
							</li>
						</ul>
					</div>
					<div class="edina_tm_list_wrap blog_list" data-column="3" data-space="30">
						<ul class="total">
							<li class="wow fadeInUp" data-wow-duration="1.2s">
									<div class="inner_list">
										<div class="image_wrap">
											<img class="small" src="research/img/genomics_datalake_ebook.png" alt="" />
											<img class="big" src="research/img/genomics_datalake_ebook.png" alt="" width="50%"/>
											<!--<div class="news_image" data-url="research/img/strainhub.png"></div>-->
											<!--<a class="link_news" href="index.html"></a>-->
										</div>
										<div class="definitions_wrap">
											<div class="date_wrap">
												<p>Genomics, Distributed Computing, Data Engineering</p>
											</div>
											<div class="title_holder">
												<h3>Genomics Data Lake eBook</h3>
											</div>
											<div class="definition">
												<p>An eBook designed to help bioinformatics teams scale up their genomics research in the Azure cloud.</p>
											</div>
											<div class="full_def">
													<p>Centralizing your data in a data lake has the potential to help scale and automate bioinformatics pipelines (including secondary and tertiary analyses and machine learning) in cloud. Azure Data Lake is touted as a limitless service place for storing your data. It provides the ability to store and organize petabyte-size files and connect to distributed computing resources (like Azure Databricks) with ease. In addition, Data Lake offers enterprise-grade security and role-based access controls.</p>
													<br>
													<p>In this book, we discuss the utility of Azure Data Lake and how this flexible and scalable storage option promotes collaboration and scalability in your genomics practice while also ensuring a secure and stable environment for your genomics data. Plus, with its easy integration with other Azure services, orchestrating and automating data movements and bioinformatics pipelines has never been easier (or faster).</p>
											</div>
											<div class="edina_tm_popup_share_wrap">
													<ul>
														<li><a href="https://zenodo.org/record/4474520" target="_blank">eBook Download</a></li>
														<!-- <li><a href="https://www.bluegranite.com/genomics-data-lake-ebook" target="_blank">BlueGranite eBook Page</a></li> -->
													</ul>
												</div>
											<div class="read_more">
												<a href="#"><span>Read More</span></a>
											</div>
										</div>
									</div>
								</li>
							<li class="wow fadeInUp" data-wow-duration="1.2s">
								<div class="inner_list">
									<div class="image_wrap">
										<img class="small" src="research/img/pf_csp_haplotypes.png" alt="" />
										<img class="big" src="research/img/pf_csp_haplotypes.png" alt="" width="50%"/>
									</div>
									<div class="definitions_wrap">
										<div class="date_wrap">
											<p>Genomics, Phylogenetics, Infectious Diseases</p>
										</div>
										<div class="title_holder">
											<h3><i>P. falciparum</i> CSP Haploytyping</h3>
										</div>
										<div class="definition">
											<p>Modeling PfCSP haplotypes and their changes in protein interaction with human immunological proteins.</p>
										</div>
										<div class="full_def">
											<p>The world's first malaria vaccine RTS,S provides only partial protection against <i>Plasmodium falciparum</i> infections. The explanation for such low efficacy is unclear. This study examined the associations of parasite genetic variations with binding affinity to human immunological proteins including human leukocyte antigen (HLA) and T cell receptors (TCR) involved in RTS,S-induced immune responses. Multiplicity of infections was determined by amplicon deep sequencing of merozoite surface protein 1 (PfMSP1). Genetic variations in the C-terminal of circumsporozoite protein (PfMSP1) gene were examined across 88 samples of <i>P. falciparum</i> collected from high and low transmission settings of Ghana. Binding interactions of PfMSP1 variants and HLA/TCR were analyzed using NetChop} and HADDOCK predictions. Anti-CSP IgG levels were measured by ELISA in a subset of 10 samples. High polyclonality was detected among <i>P. falciparum</i> infections. A total 27 CSP haplotypes were detected among samples. A significant correlation was detected between the CSP and MSP multiplicity of infection (MOI). No clear clustering of haplotypes was observed by geographic regions. The number of genetic differences in PfCSP between 3D7 and non-3D7 variants does not influence binding interactions to HLA/T cells nor anti-CSP IgG levels. Nevertheless, PfCSP peptide length significantly affects its molecular weight and binding affinity to the HLA. The presence of multiple non-3D7 strains among <i>P. falciparum</i> infections in Ghana impact the effectiveness of RTS,S. Longer PfCSP peptides may elicit a stronger immune response and should be considered in future version RTS,S. The molecular mechanisms of RTS,S cell-mediated immune responses related to longer CSP peptides warrants further investigations.</p>
										</div>
										<div class="edina_tm_popup_share_wrap">
												<ul>
													<li><a href="https://www.sciencedirect.com/science/article/pii/S1567134823000163" target="_blank">Paper</a></li>
													<!-- <li><a href="https://www.biorxiv.org/content/10.1101/2021.12.29.474460" target="_blank">Preprint</a></li> -->
													<li><a href="https://github.com/colbyford/Ghana_CSP_Haplotypes" target="_blank">GitHub</a></li>
													<li><a href="research/PfCSP_ASTMH2020.pdf" target="_blank">ASTMH 2020 Poster</a></li>
												</ul>
											</div>
										<div class="read_more">
											<a href="#"><span>Read More</span></a>
										</div>
									</div>
								</div>
							</li>
							<li class="wow fadeInUp" data-wow-duration="1.2s">
								<div class="inner_list">
									<div class="image_wrap">
										<img class="small" src="research/img/sars-cov-2-variants.png" alt="" />
										<img class="big" src="research/img/sars-cov-2-variants.png" alt="" width="50%"/>
									</div>
									<div class="definitions_wrap">
										<div class="date_wrap">
											<p>Genomics, Infectious Diseases, Bioinformatics</p>
										</div>
										<div class="title_holder">
											<h3>SARS-CoV-2 Variant Tracking</h3>
										</div>
										<div class="definition">
											<p>Building a visual dashboard for cataloging SARS-CoV-2 variants geographically.</p>
										</div>
										<div class="full_def">
											<p>Several new variants of the SARS-CoV-2 have been isolated in the United States, Mexico, and Canada. Many of the variants contain single variants of functional significance (e.g. S: N501Y increases transmissibility). To study the occurrence and co-circulation of these variants, we have developed an easy-to-use dashboard.</p>
										</div>
										<div class="edina_tm_popup_share_wrap">
												<ul>
													<li><a href="https://janieslab.github.io/sars-cov-2" target="_blank">Dashboard</a></li>
													<li><a href="https://github.com/janieslab/SARS-CoV-2_VariantDashboard" target="_blank">GitHub</a></li>
													<li><a href="https://www.medrxiv.org/content/10.1101/2021.01.27.21250648v1" target="_blank">medRxiv Preprint</a></li>
												</ul>
											</div>
										<div class="read_more">
											<a href="#"><span>Read More</span></a>
										</div>
									</div>
								</div>
							</li>
						</ul>
					</div>
					<div class="edina_tm_list_wrap blog_list" data-column="3" data-space="30">
						<ul class="total">
							<li class="wow fadeInUp" data-wow-duration="1.2s">
									<div class="inner_list">
										<div class="image_wrap">
											<img class="small" src="research/img/sparkitecture.png" alt="" />
											<img class="big" src="research/img/sparkitecture.png" alt="" width="50%"/>
											<!--<div class="news_image" data-url="research/img/strainhub.png"></div>-->
											<!--<a class="link_news" href="index.html"></a>-->
										</div>
										<div class="definitions_wrap">
											<div class="date_wrap">
												<p>Machine Learning, Distributed Computing, Data Engineering</p>
											</div>
											<div class="title_holder">
												<h3>Sparkitecture</h3>
											</div>
											<div class="definition">
												<p>A collection of "cookbook-style" scripts for simplifying data engineering and machine learning in Apache Spark.</p>
											</div>
											<div class="full_def">
													<p>Apache Spark is a highly-scalable, massively-parallel computing platform perfect for machine learning and data engineering tasks. Using distributed processing with the Spark API, users can perform various tasks on huge amounts of data using their their preferred language (Python, R, Scala, SQL, etc.), but often there is a bit of a learning curve to using the Spark functionality (PySpark or SparkR) even if the user is a pro at the base language. Sparkitecture is a ebook collection of various script to help make this process a little easier.</p>
											</div>
											<div class="edina_tm_popup_share_wrap">
													<ul>
														<li><a href="https://www.sparkitecture.io/" target="_blank">Sparkitecture.io</a></li>
														<li><a href="https://github.com/colbyford/sparkitecture" target="_blank">GitHub</a></li>
														<!--<li><a href="" target="_blank">Paper Coming Soon!</a></li>-->
														<!--<li><a href="https://www.biorxiv.org/content/10.1101/650283v1" target="_blank">Paper</a></li>-->
													</ul>
												</div>
											<div class="read_more">
												<a href="#"><span>Read More</span></a>
											</div>
										</div>
									</div>
								</li>

							<li class="wow fadeInUp" data-wow-duration="1.2s">
									<div class="inner_list">
										<div class="image_wrap">
											<img class="small" src="research/img/malariaml.png" alt="" />
											<img class="big" src="research/img/malariaml.png" alt="" width="50%"/>
											<!--<div class="news_image" data-url="research/img/strainhub.png"></div>-->
											<!--<a class="link_news" href="index.html"></a>-->
										</div>
										<div class="definitions_wrap">
											<div class="date_wrap">
												<p>Genomics, Machine Learning, Infectious Diseases</p>
											</div>
											<div class="title_holder">
												<h3>Machine Learning in Malaria</h3>
											</div>
											<div class="definition">
												<p>Machine Learning Modeling in the Prediction of Artemisinin Resistance and Diagnostic Test Sensitivity in Malaria.</p>
											</div>
											<div class="full_def">
											<h4 style="color:white">Parallel Processing and Ensemble Machine Learning Modeling for the Prediction of Artemisinin Resistance in Malaria (Malaria DREAM Challenge 2019 Submission).</h4>
													<p>The Malaria DREAM Challenge is open to anyone interested in contributing to the development of computational models that address important problems in advancing the fight against malaria. The overall goal of the first Malaria DREAM Challenge is to predict Artemisinin (Art) drug resistance level of a test set of malaria parasites using their in vitro transcription data and a training set consisting of published in vivo and unpublished in vitro transcriptomes. The in vivo dataset consists of ~1000 transcription samples from various geographic locations covering a wide range of life cycles and resistance levels, with other accompanying data such as patient age, geographic location, Art combination therapy used, etc. [Mok et al., (2015) Science]. The in vitro transcription dataset consists of 55 isolates, with transcription collected at two timepoints (6 and 24 hours post-invasion), in the absence or presence of an Art perturbation, for two biological replicates using a custom microarray at the Ferdig lab. Using these transcription datasets, participants will be asked to predict three different resistance states of a subset of the 55 in vitro isolate samples.</p>
											</div>
											<div class="edina_tm_popup_share_wrap">
													<ul>
														<li><a href="https://www.synapse.org/#!Synapse:syn16924919/wiki/583955" target="_blank">DREAM Challenge Link</a></li>
														<li><a href="https://github.com/colbyford/malaria_DREAM2019" target="_blank">GitHub</a></li>
														<li><a href="https://f1000research.com/articles/9-62" target="_blank">Paper</a></li>
													</ul>
												</div>
											<div class="read_more">
												<a href="#"><span>Read More</span></a>
											</div>

											<div class="full_def">
											<br><hr>
											<h4 style="color:white">Modeling <i>Plasmodium falciparum</i> Diagnostic Test Sensitivity using Machine Learning with Histidine-Rich Protein 2 Variants.</h4>
													<p>Malaria, predominantly caused by <i>Plasmodium falciparum</i>, poses one of largest and most durable health threats in the world. Previously, simplistic regression-based models have been created to characterize malaria infections, though these models often only include a couple genetic factors. Specifically, the Baker et al., 2005 model uses two types of particular repeats in histidine-rich protein 2 (PfHRP2) to assert <i>P. falciparum</i> infection, though the efficacy of this model has waned over recent years due to genetic mutations in the parasite. In this work, we use a dataset of 406 <i>P. falciparum</i> PfHRP2 genetic sequences collected in Ethiopia and derived a larger set of motif repeat matches for use in generating a series of diagnostic machine learning models. Here we show that the usage of additional and different motif repeats proves effective in predicting infection. Furthermore, we use machine learning model explanability methods to highlight which of the repeat types are most important, thereby suggesting potential targets for future versions of rapid diagnostic tests.</p>
											</div>
											<div class="edina_tm_popup_share_wrap">
													<ul>
														<li><a href="https://github.com/colbyford/pfHRP_MLModel" target="_blank">GitHub</a></li>
														<!-- <li><a href="https://www.medrxiv.org/content/10.1101/2020.05.27.20114785v2" target="_blank">Paper</a></li> -->
														<li><a href="https://www.frontiersin.org/articles/10.3389/fitd.2021.707313/full" target="_blank">Paper</a></li>
													</ul>
												</div>
										</div>
									</div>
								</li>


							<!--<li class="wow fadeInUp" data-wow-duration="1.2s">
								<div class="inner_list">
									<div class="image_wrap">
										<img class="small" src="research/img/pvivax_positive_selection.png" alt="" />
										<img class="big" src="research/img/pvivax_positive_selection.png" alt="" width="50%"/>
									</div>
									<div class="definitions_wrap">
										<div class="date_wrap">
											<p>Genomics, Infectious Diseases</p>
										</div>
										<div class="title_holder">
											<h3><i>P. vivax</i> EBP Positive Selection Analysis</h3>
										</div>
										<div class="definition">
												<p>Positive selection analysis of the erythrocyte binding protein in <i>P. vivax</i> as it relates to malaria infections.</p>
										</div>
										<div class="edina_tm_popup_share_wrap">
												<ul>
													<li><a href="https://public.tableau.com/profile/cford38#!/vizhome/P_vivax-PositiveSelection/P_vivaxAnalysis" target="_blank">Dashboard</a></li>
													<li><a href="#">Paper (Coming Soon!)</a></li>
												</ul>
											</div>
										<div class="read_more">
											<a href="#"><span>Read More</span></a>
										</div>
									</div>
								</div>
							</li>-->

							<li class="wow fadeInUp" data-wow-duration="1.2s">
								<div class="inner_list">
									<div class="image_wrap">
										<img class="small" src="research/img/strainhub.png" alt="" />
										<img class="big" src="research/img/strainhub.png" alt="" width="50%"/>
										<!--<div class="news_image" data-url="research/img/strainhub.png"></div>-->
										<!--<a class="link_news" href="index.html"></a>-->
									</div>
									<div class="definitions_wrap">
										<div class="date_wrap">
											<p>Genomics, Phylogenetics, Infectious Diseases</p>
										</div>
										<div class="title_holder">
											<h3>StrainHub</h3>
										</div>
										<div class="definition">
											<p>Shiny web application for visualizing disease transmittion networks from phylogenetic trees.</p>
										</div>
										<div class="full_def">
												<p>Strainhub is designed as a web-based software to generate disease transmission networks and associated metrics from a combination of a phylogenetic tree and a metadata associated file. The software maps the metadata onto the tree and performs a parsimony ancestry reconstruction step to create links between the associated metadata and enable the construction of the network. </p>
										</div>
										<div class="edina_tm_popup_share_wrap">
												<ul>
													<li><a href="https://strainhub.io" target="_blank">Web App</a></li>
													<li><a href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btz646/5550625?redirectedFrom=fulltext" target="_blank">Paper</a></li>
													<!--<li><a href="https://www.biorxiv.org/content/10.1101/650283v1" target="_blank">Paper</a></li>-->
													<li><a href="https://github.com/colbyford/StrainHub/tree/master/pkg" target="_blank">R Package</a></li>
												</ul>
											</div>
										<div class="read_more">
											<a href="#"><span>Read More</span></a>
										</div>
									</div>
								</div>
							</li>
						</ul>
					</div>
					<div class="edina_tm_list_wrap blog_list" data-column="3" data-space="30">
									<ul class="total">
							<li class="wow fadeInUp" data-wow-duration="1.2s">
								<div class="inner_list">
									<div class="image_wrap">
										<img class="small" src="research/img/ecoli_genetic_capitalism.png" alt="" />
										<img class="big" src="research/img/ecoli_genetic_capitalism.png" alt="" width="50%"/>
										<!--<div class="news_image" data-url="research/img/ecoli_genetic_capitalism.png"></div>-->
										<!--<a class="link_news" href="index.html"></a>-->
									</div>
									<div class="definitions_wrap">
										<div class="date_wrap">
											<p>Genomics, Infectious Diseases</p>
										</div>
										<div class="title_holder">
											<h3>Genetic Capitalism in <i>E. coli</i></h3>
										</div>
										<div class="definition">
												<p>Persistence of Genes that Confer Antimicrobial Resistance in the History of <i>Escherichia coli</i> Genomes</p>
										</div>
										<div class="full_def">
											<p>Antimicrobial resistance (AMR) in pathogenic strains of bacteria, such as <i>Escherichia coli</i> (<i>E. coli</i>), adversely impact personal and public health. In this study, we examine competing hypotheses for the evolution of AMR including: 1) "genetic capitalism" in which genotypes that confer antibiotic resistance are gained and not often lost in lineages, and 2) "stabilizing selection" in which genotypes that confer antibiotic resistance are gained and lost often. To test these hypotheses, we assembled a dataset that includes annotations for 409 AMR genotypes and a phylogenetic tree based on genome-wide single nucleotide polymorphisms from 29,255 isolates of <i>E. coli</i>. We used phylogenetic methods to count the times each AMR genotype was gained and lost across the tree and used model-based clustering of the genotypes with respect to their gain and loss rates. We demonstrate that many genotypes cluster to support the hypothesis for genetic capitalism while a few cluster to support the hypothesis for stabilizing selection. Comparing the sets of genotypes that fall under each of the hypotheses, we found a statistically significant difference in the breakdown of resistance mechanisms through which the AMR genotypes function. The result that many AMR genotypes cluster under genetic capitalism reflects that strong positive selective forces, primarily induced by human industrialization of antibiotics, outweigh the potential fitness costs to the bacterial lineages for carrying the AMR genotypes. We expect genetic capitalism to further drive bacterial lineages to resist antibiotics. We find that antibiotics that function via replacement and efflux tend to behave under stabilizing selection and thus may be valuable in an antibiotic cycling strategy.</p>
										</div>
										<div class="edina_tm_popup_share_wrap">
												<ul>
													<li><a href="https://public.tableau.com/profile/cford38#!/vizhome/E_coliGenotypeSetsViz/E_coliGenotypeChanges" target="_blank">Dashboard</a></li>
													<li><a href="https://github.com/supramap/ecoli_amr_persistence/tree/master/Genetic_Capitalism" target="_blank">GitHub</a></li>
													<li><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/cla.12421" target="_blank">Paper (Cladistics)</a></li>
												</ul>
											</div>
										<div class="read_more">
											<a href="#"><span>Read More</span></a>
										</div>
									</div>
								</div>
							</li>
							<!-- <li class="wow fadeInUp" data-wow-duration="1.2s">
									<div class="inner_list">
										<div class="image_wrap">
											<img class="small" src="research/img/sourcerr.png" alt="" />
											<img class="big" src="research/img/sourcerr.png" alt="" width="50%"/> -->
											<!--<div class="news_image" data-url="research/img/sourcerr.png"></div>-->
											<!--<a class="link_news" href="index.html"></a>-->
										<!-- </div>
										<div class="definitions_wrap">
											<div class="date_wrap">
												<p>R Package</p>
											</div>
											<div class="title_holder">
												<h3>sourcerr</h3>
											</div>
											<div class="definition">
													<p>Easily output package citations and tidy session information for reproducibility in research publications.</p>
											</div>
											<div class="edina_tm_popup_share_wrap">
													<ul>
														<li><a href="https://github.com/colbyford/sourcerr" target="_blank">GitHub</a></li>
													</ul>
												</div>
											<div class="read_more">
												<a href="#"><span>Read More</span></a>
											</div>
										</div>
									</div>
								</li> -->

								<li class="wow fadeInUp" data-wow-duration="1.2s">
										<div class="inner_list">
											<div class="image_wrap">
												<img class="small" src="research/img/msgen.png" alt="" />
												<img class="big" src="research/img/msgen.png" alt="" width="50%"/>
												<!--<div class="news_image" data-url="research/img/msgen.png"></div>-->
												<!--<a class="link_news" href="index.html"></a>-->
											</div>
											<div class="definitions_wrap">
												<div class="date_wrap">
													<p>R Package, Genomics</p>
												</div>
												<div class="title_holder">
													<h3>msgen</h3>
												</div>
												<div class="definition">
														<p>R functions for interfacing with the Microsoft Genomics service in Azure.</p>
												</div>
												<div class="full_def">
													<p>The Microsoft Genomics service in Azure can power genome sequencing using a cloud implementation of the Burrows-Wheeler Aligner (BWA) and the Genome Analysis Toolkit (GATK) for secondary analysis. The pipeline can take in multiple FASTQ and BAM files and provides alignment and variant outputs. The msgen package provides an interface to use the service from within R.</p>
												</div>
												<div class="edina_tm_popup_share_wrap">
														<ul>
															<li><a href="https://github.com/colbyford/msgen" target="_blank">GitHub</a></li>
															<li><a href="https://colbyford.medium.com/msgen-r-package-af752f5ef3eb" target="_blank">Blog Post</a></li>
														</ul>
													</div>
												<div class="read_more">
													<a href="#"><span>Read More</span></a>
												</div>
											</div>
										</div>
									</li>

										<!-- <li class="wow fadeInUp" data-wow-duration="1.2s">
											<div class="inner_list">
												<div class="image_wrap">
													<img class="small" src="research/img/quilt.png" alt="" />
													<img class="big" src="research/img/quilt.png" alt="" width="50%"/>
													<\!--<div class="news_image" data-url="research/img/quilt.png"></div>--\>
													<\!--<a class="link_news" href="index.html"></a>--\>
												</div>
												<div class="definitions_wrap">
													<div class="date_wrap">
														<p>R Package</p>
													</div>
													<div class="title_holder">
														<h3>quilt</h3>
													</div>
													<div class="definition">
															<p>A patchwork of efficient and tidy multidimensional data operations.</p>
													</div>
													<div class="edina_tm_popup_share_wrap">
															<ul>
																<li><a href="https://github.com/colbyford/quilt" target="_blank">GitHub</a></li>
															</ul>
														</div>
													<div class="read_more">
														<a href="#"><span>Read More</span></a>
													</div>
												</div>
											</div>
										</li> -->
										<li class="wow fadeInUp" data-wow-duration="1.2s">
												<div class="inner_list">
													<div class="image_wrap">
														<img class="small" src="research/img/BantuMigration.png" alt="" />
														<img class="big" src="research/img/BantuMigration.png" alt="" width="50%"/>
													<!--<div class="news_image" data-url="research/img/BantuMigration.png"></div>-->
														<!--<a class="link_news" href="index.html"></a>-->
													</div>
													<div class="definitions_wrap">
														<div class="date_wrap">
															<p>Human Genomics, Phylogenetics</p>
														</div>
														<div class="title_holder">
															<h3>An Integrated Phylogeographic Analysis of the Bantu Migration</h3>
														</div>
														<div class="definition">
																<p>Using phylogenetic analysis, dimensionality reduction, and machine learning, a combined migratory model of the Bantu migration was formed with heterogeneous data.</p>
														</div>
														<div class="edina_tm_popup_share_wrap">
																<ul>
																	<li><a href="https://github.com/colbyford/BantuMigration" target="_blank">GitHub</a></li>
																	<li><a href="https://public.tableau.com/profile/cford38#!/vizhome/BantuMigration/AnIntegratedPhylogeographicAnalysisoftheBantuMigration" target="_blank">Dashboard</a></li>
																	<li><a href="https://pqdtopen.proquest.com/doc/2036377441.html?FMT=ABS" target="_blank">Paper</a></li>
																</ul>
															</div>
														<div class="read_more">
															<a href="#"><span>Read More</span></a>
														</div>
													</div>
												</div>
											</li>
										</ul>
									</div>
									<div class="edina_tm_list_wrap blog_list" data-column="3" data-space="30">
													<ul class="total">
											<li class="wow fadeInUp" data-wow-duration="1.2s">
													<div class="inner_list">
														<div class="image_wrap">
															<img class="small" src="research/img/parEBEN.png" alt="" />
															<img class="big" src="research/img/parEBEN.png" alt="" width="50%"/>
															<!--<div class="news_image" data-url="research/img/parEBEN.png"></div>-->
															<!--<a class="link_news" href="index.html"></a>-->
														</div>
														<div class="definitions_wrap">
															<div class="date_wrap">
																<p>R Package, Genomics</p>
															</div>
															<div class="title_holder">
																<h3>parEBEN</h3>
															</div>
															<div class="definition">
																	<p>Parallel Implementations of the Empirical Bayesian Elastic Net Cross-Validation in R.</p>
															</div>
															<div class="full_def">
																<p>The Empirical Bayesian Elastic Net (EBEN) algorithm was developed by Huang et al. for handling multicollinearity in generalized linear regression models. Historically, this has been used in the analysis of quantitative trait loci (QTLs) and gene-gene interactions (epistasis). In addition to the algorithm, the group also created the EBEN package for R. This package includes functions to generate the elastic nets for both binomial and gaussian priors. These functions are efficient and do not require large amounts of computational time. However, the package also includes functions for the cross-validation of those models. While essential, this step is a considerably more complex task. The cross-validation functions perform a sweep to determine hyperparameters and minimize prediction error. More specifically, an n-fold cross-validation sweep is performed to minimize error by trying combinations of two parameters (α and λ) in a stepped manner. Experimentally, it has been shown that this can take a rather extended amount of time, especially on larger datasets (as seen in genomics problems).</p>
															</div>
															<div class="edina_tm_popup_share_wrap">
																	<ul>
																		<li><a href="https://github.com/colbyford/parEBEN" target="_blank">GitHub</a></li>
																		<li><a href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btaa216/5813727" target="_blank">Paper</a></li>
																	</ul>
																</div>
															<div class="read_more">
																<a href="#"><span>Read More</span></a>
															</div>
														</div>
													</div>
												</li>

													<li class="wow fadeInUp" data-wow-duration="1.2s">
														<div class="inner_list">
															<div class="image_wrap">
																<img class="small" src="research/img/languageresearch.png" alt="" />
																<img class="big" src="research/img/languageresearch.png" alt="" width="50%"/>
																<!--<div class="news_image" data-url="research/img/languageresearch.png"></div>-->
																<!--<a class="link_news" href="index.html"></a>-->
															</div>
															<div class="definitions_wrap">
																<div class="date_wrap">
																	<p>Human Genomics, Phylogenetics, Linguistics</p>
																</div>
																<div class="title_holder">
																	<h3>Visualizing Linguistic Disparity of Uto-Aztecan Languages and Bantu Languages</h3>
																</div>
																<div class="definition">
																		<p>Using unsupervised learning for dimensionality reduction and visualization.</p>
																</div>
																<div class="edina_tm_popup_share_wrap">
																		<ul>
																			<li><a href="research/languageresearch/index.html" target="_blank">Dashboard</a></li>
																		</ul>
																	</div>
																<div class="read_more">
																	<a href="#"><span>Read More</span></a>
																</div>
															</div>
														</div>
													</li>
													<li class="wow fadeInUp" data-wow-duration="1.2s">
															<div class="inner_list">
																<div class="image_wrap">
																	<img class="small" src="research/img/mers.png" alt="" />
																	<img class="big" src="research/img/mers.png" alt="" width="50%"/>
																	<!--<div class="news_image" data-url="research/img/mers.png"></div>-->
																	<!--<a class="link_news" href="index.html"></a>-->
																</div>
																<div class="definitions_wrap">
																	<div class="date_wrap">
																		<p>Phylogenetics, Epidemiology, Infectious Diseases</p>
																	</div>
																	<div class="title_holder">
																		<h3>Spread of Middle East Respiratory Coronavirus: Genetic versus Epidemiological Data</h3>
																	</div>
																	<div class="definition">
																			<p>Analysis and visualzation of the WHO reports of MERS infections versus genomic data.</p>
																	</div>
																	<div class="edina_tm_popup_share_wrap">
																			<ul>
																				<li><a href="http://journals.uic.edu/ojs/index.php/ojphi/article/view/7581" target="_blank">Paper</a></li>
																			</ul>
																		</div>
																	<div class="read_more">
																		<a href="#"><span>Read More</span></a>
																	</div>
																</div>
															</div>
														</li>
													</ul>
												</div>
												<div class="edina_tm_list_wrap blog_list" data-column="3" data-space="30">
																<ul class="total">
														<li class="wow fadeInUp" data-wow-duration="1.2s">
																<div class="inner_list">
																	<div class="image_wrap">
																		<img class="small" src="research/img/mpsiiib.png" alt="" />
																		<img class="big" src="research/img/mpsiiib.png" alt="" width="50%"/>
																		<!--<div class="news_image" data-url="research/img/mpsiiib.png"></div>-->
																		<!--<a class="link_news" href="index.html"></a>-->
																	</div>
																	<div class="definitions_wrap">
																		<div class="date_wrap">
																			<p>Human Genomics, Machine Learning</p>
																		</div>
																		<div class="title_holder">
																			<h3>MPS-IIIB/NAGLU Prediction</h3>
																		</div>
																		<div class="definition">
																				<p>PolyPhen2 + Machine Learning prediction of the effects of genetic mutations on mucopolysaccharidosis IIIB.</p>
																		</div>
																		<div class="edina_tm_popup_share_wrap">
																				<ul>
																					<li><a href="https://github.com/colbyford/NAGLU" target="_blank">GitHub</a></li>
																					<!--<li><a href="research/mpsiiib/NAGLU_FinalReport_FordNodzakUppalZhao.pdf" target="_blank">Paper</a></li>-->
																					<li><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/humu.23875" target="_blank">Paper</a></li>
																					<li><a href="https://www.biorxiv.org/content/10.1101/598870v1" target="_blank">bioRxiv Paper</a></li>

																				</ul>
																			</div>
																		<div class="read_more">
																			<a href="#"><span>Read More</span></a>
																		</div>
																	</div>
																</div>
															</li>
						</ul>
					</div>
				</div>
			</div>
		</div>
		<!-- /NEWS -->

		<!-- PARTNERS -->
		<div class="edina_tm_section">
			<div class="edina_tm_partners_wrap_total">
				<div class="edina_tm_universal_box_wrap">
					<div class="bg_wrap">
						<div class="overlay_image partners jarallax" data-speed="0"></div>
						<div class="overlay_color partners"></div>
					</div>
					<div class="content partners">
						<div class="container">
							<div class="edina_tm_partners_wrap">
								<ul class="owl-carousel">
									<li class="item">
										<img src="img/partners/uncc_c_white.png" alt="" />
									</li>
									<li class="item">
										<img src="img/partners/microsoft_white.png" alt="" />
									</li>
									<li class="item">
										<img src="img/partners/databricks_white.png" alt="" />
									</li>
									<li class="item">
										<img src="img/partners/nsf_white.png" alt="" />
									</li>
									<li class="item">
										<img src="img/partners/darpa.png" alt="" />
									</li>
									<li class="item">
										<img src="img/partners/dtra_white.png" alt="" />
									</li>
								</ul>
							</div>
						</div>
					</div>
				</div>
			</div>
		</div>
		<!-- /PARTNERS -->

		<!-- WORK TOGETHER -->
		<div class="edina_tm_section" id="contact">
				<div class="edina_tm_main_title_holder_wrap contact">
						<div class="number_wrap">
								<span>04</span>
						</div>
					</div>
			<div class="container">
				<div class="edina_tm_work_together_total">
					<div class="edina_tm_work_together_wrap">
						<h3 class="wow fadeIn" data-wow-duration="1.2s">Let's Connect!</h3>
						<p class="wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.2s">Want to collaborate, connect, or a consult? Let's chat and see how we can work together to do awesome things...</p>
						<div class="invite wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.4s">
								<a href="mailto:colby.ford@uncc.edu" title="Email" target="_blank">Send Me an Email</a>
						</div>
					</div>
				</div>
			</div>
		</div>
		<!-- /WORK TOGETHER -->

		<!-- CONTACT -->
		<!--<div class="edina_tm_section" id="contact">
			<div class="edina_tm_main_title_holder_wrap contact">
				<div class="number_wrap">
					<span>04</span>
				</div>
				<div class="title_wrap">
					<span>Contact Me</span>
				</div>
			</div>
			<div class="edina_tm_contact_wrap">
				<div class="short_info">
					<div class="container">
						<div class="subtitle">
							<p class="wow fadeIn" data-wow-duration="1.2s">Any question? Reach out to me and I will get back to you shortly.</p>
						</div>
					</div>
				</div>
				<div class="main_input_wrap">
					<form action="/" method="post" class="contact_form" id="contact_form">
						<div class="returnmessage" data-success="Your message has been received, We will contact you soon."></div>
						<div class="empty_notice"><span>Please Fill Required Fields</span></div>
						<div class="wrap wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.2s">
							<input id="name" type="text" placeholder="Your Name">
						</div>
						<div class="wrap wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.4s">
							<input id="email" type="text" placeholder="Your Email">
						</div>
						<div class="wrap wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.6s">
							<textarea id="message" placeholder="Your Message"></textarea>
						</div>
						<div class="edina_tm_button wow fadeIn" data-wow-duration="1.2s" data-wow-delay="0.8s">
							<a id="send_message" href="#">Send Message</a>
						</div>
					</form>
				</div>
			</div>
		</div>-->
		<!-- /CONTACT -->

		<!-- FOOTER -->
		<div class="edina_tm_section">
			<div class="edina_tm_footer_wrap">
				<p class="wow fadeIn"  data-wow-duration="1.2s">Copyright &copy; 2024 Colby T. Ford, Ph.D. All Rights Reserved.<br>Theme by <a href="https://themeforest.net/user/marketify">Marketify</a></p>
			</div>
		</div>
		<!-- /FOOTER -->

	</div>
	<!-- /CONTENT -->

  	<!-- TOP TOP -->
	<div class="edina_tm_to_top_wrap">
 		<a href="#">Back To Top</a>
	</div>
 	<!-- /TO TOP -->

</div>
<!-- / WRAPPER ALL -->

<!-- SCRIPTS -->
<script src="js/jquery.js"></script>
<!--[if lt IE 10]> <script type="text/javascript" src="js/ie8.js"></script> <![endif]-->
<script src="js/plugins.js"></script>
<script src="js/init.js"></script>
<!-- /SCRIPTS -->

</body>
</html>
